flyctl launch panics while building empty Dockerfiles

`flyctl launch` panics when attempting to build "empty" Dockerfiles. However, running `flyctl deploy` appears to successfully build the image and does not panic.

```text
panic: interface conversion: interface {} is nil, not *pflag.FlagSet
```

- Operating system: `Fedora Linux 36 (Workstation Edition)`
- fly version: `flyctl v0.0.331 linux/amd64 Commit: 95563e1 BuildDate: 2022-06-02T22:18:16Z`

<hr>

# Steps to reproduce
1. Create a new directory with the following Dockerfile:
```text
# Dockerfile
FROM scratch
WORKDIR /
```
2. Run `flyctl launch` within that directory

# Observations
- Both the initial and all subsequent calls to `flyctl launch` trigger the panic
- Saying "No" at the `Would you like to deploy now?` prompt and immediately running `flyctl deploy` will not trigger the panic

# Example command output
**flyctl launch**
```text
$ flyctl launch
An existing fly.toml file was found for app bitter-field-7009
App is not running, deploy...
Deploying bitter-field-7009
==> Validating app configuration
--> Validating app configuration done
Services
TCP 80/443 ⇢ 8080
Remote builder fly-builder-falling-dawn-2803 ready
==> Creating build context
--> Creating build context done
==> Building image with Docker
--> docker host: 20.10.12 linux x86_64
Sending build context to Docker daemon     660B
panic: interface conversion: interface {} is nil, not *pflag.FlagSet

goroutine 118 [running]:
github.com/superfly/flyctl/internal/flag.FromContext(...)
	/home/runner/work/flyctl/flyctl/internal/flag/context.go:19
github.com/superfly/flyctl/internal/flag.GetBool({0x16e9a28, 0xc000991b00}, {0x1485a21, 0x5})
	/home/runner/work/flyctl/flyctl/internal/flag/context.go:70 +0x94
github.com/superfly/flyctl/internal/build/imgsrc.runBuildKitBuild.func4.1(0xc000b32200, {0x16e9a28, 0xc000991b00}, 0xc000815a40, 0x94, 0xc0009efe00, 0xc000b2db80, 0xc000a57400)
	/home/runner/work/flyctl/flyctl/internal/build/imgsrc/dockerfile_builder.go:321 +0x2e5
github.com/superfly/flyctl/internal/build/imgsrc.runBuildKitBuild.func4()
	/home/runner/work/flyctl/flyctl/internal/build/imgsrc/dockerfile_builder.go:371 +0x2f0
golang.org/x/sync/errgroup.(*Group).Go.func1()
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:57 +0x67
created by golang.org/x/sync/errgroup.(*Group).Go
	/home/runner/go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:54 +0x92
```

**flyctl deploy**
```text
$ flyctl deploy
==> Verifying app config
--> Verified app config
==> Building image
Remote builder fly-builder-falling-dawn-2803 ready
==> Creating build context
--> Creating build context done
==> Building image with Docker
--> docker host: 20.10.12 linux x86_64
Sending build context to Docker daemon     660B
[+] Building 0.0s (3/3) FINISHED                                                                                                                                      
 => CACHED [internal] load remote build context                                                                                                                  0.0s
 => CACHED copy /context /                                                                                                                                       0.0s
 => exporting to image                                                                                                                                           0.0s
 => => writing image sha256:47ccfa9dbc8fce746abe2416138bdb2c18d6ade5c7d3ba54ccfcab5bc587e2b3                                                                     0.0s
 => => naming to registry.fly.io/bitter-field-7009:deployment-1654227846                                                                                         0.0s
--> Building image done
==> Pushing image to fly
The push refers to repository [registry.fly.io/bitter-field-7009]
deployment-1654227846: digest: sha256:1a0dbf7c537f02dc0091e3abebae0ccac83da6aa147529f5de49af0f23cd9e8e size: 313
--> Pushing image done
image: registry.fly.io/bitter-field-7009:deployment-1654227846
image size: 0 B
==> Creating release
--> release v2 created

--> You can detach the terminal anytime without stopping the deployment
==> Monitoring deployment

 1 desired, 1 placed, 0 healthy, 1 unhealthy
--> v0 failed - Failed due to unhealthy allocations - no stable job version to auto revert to and deploying as v1 

--> Troubleshooting guide at https://fly.io/docs/getting-started/troubleshooting/
Error abort
```

**docker build**
```text
docker build -t fly-panic .
STEP 1/2: FROM scratch
STEP 2/2: WORKDIR /
COMMIT fly-panic
--> 91e67b10201
Successfully tagged localhost/fly-panic:latest
91e67b102016d12cc09db54ee63704a06f2b28cd97afbcc7d97fc441b4289c90
```

**docker run**
```text
docker run --rm fly-panic
Error: no command or entrypoint provided, and no CMD or ENTRYPOINT from image
```

--- a/internal/build/imgsrc/dockerfile_builder.go
+++ b/internal/build/imgsrc/dockerfile_builder.go
@@ -24,7 +24,6 @@ import (
 	"github.com/pkg/errors"
 	"github.com/superfly/flyctl/helpers"
 	"github.com/superfly/flyctl/internal/cmdfmt"
-	"github.com/superfly/flyctl/internal/flag"
 	"github.com/superfly/flyctl/internal/render"
 	"github.com/superfly/flyctl/pkg/iostreams"
 	"github.com/superfly/flyctl/terminal"
@@ -318,7 +317,7 @@ func runBuildKitBuild(ctx context.Context, streams *iostreams.IOStreams, docker 
 			termFd, isTerm := term.GetFdInfo(os.Stderr)
 			tracer := newTracer()
 			var c2 console.Console
-			if io.ColorEnabled() && !flag.GetBool(ctx, "plain") {
+			if io.ColorEnabled() {
 				if cons, err := console.ConsoleFromFile(os.Stderr); err == nil {
 					c2 = cons
 				}

--- a/internal/command/deploy/deploy.go
+++ b/internal/command/deploy/deploy.go
@@ -88,10 +88,6 @@ func New() (cmd *cobra.Command) {
 			Name:        "nix",
 			Description: "Build with Nix",
 		},
-		flag.Bool{
-			Name:        "plain",
-			Description: "Display full Docker build output",
-		},
 	)
 
 	return

